File: 0014-longest-common-prefix/0014-longest-common-prefix.cpp
class Solution {
public:
    string longestCommonPrefix(vector<string>& strs) {
        sort(strs.begin(),strs.end());
        int n=strs[0].size();
        string ans="";
        for(int i=0;i<n;i++){
            if(strs[0][i]==strs.back()[i])
                ans.push_back(strs[0][i]);
            else    break;
        }
        return ans;
    }
};

File: 0014-longest-common-prefix/0014-longest-common-prefix.py
class Solution:
    def longestCommonPrefix(self, strs: List[str]) -> str:
        strs.sort()
        n=len(strs[0])
        ans=""
        for i in range(n):
            if strs[0][i] == strs[-1][i]:
                ans+=strs[0][i]
            else:
                break
        return ans